### ISSUE TITLE
Bootstrap - Heading levels can be wrong when parts of the interface are hidden

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1336]
If the header bar is hidden (or section titles aren't added) the heading order can be wrong - e.g. when header bar is missing the headings on the page will start at H2. It should be possible to flag this up and use the appropriate heading levels for different areas of the page depending on what is being shown. We would need to make sure that the appearance of headings remained the same regardless of whether, for example, it was an element that would normally be a H3 heading but had been promoted to H2 because of something else within the page being hidden.
